examples/olora_finetuning/README.md

```diff
@@ -39,6 +39,15 @@ OLoRA also supports quantization. To use 4-bit quantization try:
 ```bash
 python3 examples/olora_finetuning/olora_finetuning.py --base_model facebook/opt-350m --quantize
 ```
+or you can just pass a quantized model without the quantize flag.
+
+If you want to run DDP by accelerate, please run `accelerate config` to set your ddp config, and run:
+```bash
+accelerate launch examples/olora_finetuning/olora_finetuning.py --base_model facebook/opt-350m
+```
+please add `--device_map cpu` if you want to run finetune on CPU.
+
+If you want to train a quantized model like AWQ and GPTQ which do not support olora init method, please pass `--init_lora_weights gaussian`.
```

Comment:
Would it be possible to use AWQ or GPTQ right now? If the user passes `--quantize`, the usage of bitsandbytes is hard-coded.

Reply:
Yes, we can just pass a quantized model. I have updated it in the readme.